dotnet-monitor.yml

```diff
@@ -7,6 +7,7 @@ pr:
     - main
     - release/*
     - internal/release/*
+    - feature/*
```

Comment:
Do we need to do a corresponding update in GitHub to "protect" these branches (like in ADO) or are branches always protected?

Reply:
I will add branch protection for these branches.